index.html

```diff
@@ -1,6 +1,121 @@
 <!DOCTYPE html>
 <html>
-<body>
+<body> 
+    <!-- styles proposal for more neat looks -->
+    <style>
+
+            @media (min-width: 960px) {
+                .te-product-card .te-product-card-img {
+                    height: 250px;
+                }
+            }
+
+
+            .te-product-card-details {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            line-height: 30px;
+            padding: 15px 0;
+            justify-content: flex-start;
+            /* height:  100px; */
+            overflow: hidden;
+            position: relative;
+            /* text-align: left; */
+        }
+
+        .te-product-card .te-product-card-brand {
+            order: -1;
+            font-weight: normal;
+            letter-spacing: 0;
+            color: #999;
+            text-transform: uppercase;
+            font-size: 12px;
```

Comment:
I think that while applying the styles in TE project I'll leave them in `em`s and set font-size in the `.te-product-card` container. That way it will look as good as now and will be very easy to change by a POS while keeping the ratio.

Reply:
exactly! perhaps they could be different for collection and single product?

Reply:
I mean: I see a case when POS wants to change the fonts to large on the single product view, but the collections will be destroyed (as there are three products in a row and fonts are unevenly large).

Reply:
But that's it. Every POS can overwrite our CSS and make a mess, that's not up to us anymore. This only gives them a chance to control how a product card looks if they feel that our 12px is too small in their layout.

Reply:
These are two different ideas. px vs em. 
px - _more_ isolation, _less_ unexpected looks after integration of elements with POS site, _harder_ for POS to alter our changes
em - _less_ isolation, _more_ unexpected looks after integration of elements with POS site, _easier_ for POS to alter our 

Hard to make a choice.

There is another way. To serve two set of stylesheets, in px and ems :D this could be quite easily done with some PostCSS processor. POS would choose which he prefers, or we would choose for him.